ui/feedlist.js

```diff
@@ -1163,6 +1163,7 @@ export let Commands = {
 
     emptyTrash: function cmd_emptyTrash() {
         ViewList.db.query(ViewList.getQueryForView('trash-folder')).markDeleted('deleted');
+        ViewList.db.cleanupEntries();
```

Comment:
Note that `markDeleted` is async and I'm not really sure whether you'll have any guarantees on the order of these operations. Also you may want to hook into entry expiration too for the sake of users who don't empty their trash manually.